packages/devtools_app/lib/src/utils.dart

```diff
@@ -518,7 +518,14 @@ class ImmediateValueNotifier<T> extends ValueNotifier<T> {
   }
 }
 
-extension NullSafeLast<T> on List<T> {
+extension NullSafeAccess<T> on List<T> {
+  T nullSafeFirst() {
```

Comment:
nit: can be one lined as
T nullSafeFirst() => isEmpty ? null : first;

Reply:
consider also just naming "safeFirst"
Designing for null aware by default in the future, the API will be self documenting in that it will return a nullable T even when T is NNBD.

Reply:
done